modules/dr-restoring-cluster-state.adoc

```diff
@@ -211,10 +211,11 @@ Review the `NodeInstallerProgressing` status condition for etcd to verify that a
 [source,terminal]
 ----
 AllNodesAtLatestRevision
-3 nodes are at revision 3
+3 nodes are at revision 7 <1>
 ----
+<1> Confirm that this message states that all nodes are at the latest revision number, which in this example is 7.
```

Comment:
We don't usually give instructions in callouts, so I might say:
```suggestion
<1> In this example, the latest revision number is `7`.
```

Reply:
Good point. I'll adjust